### DISCUSSION TITLE
Traditional Chinese (Hong Kong) Translation

### DISCUSSION THREAD

**Discussion by sxzz** [vuejs-translations/guidelines#82] [General]
Repo: https://github.com/dy-xiaodong2022/docs-zh-hant

@dy-xiaodong2022 is looking to add a traditional Chinese version. However, according to the [Translation Guidelines](https://github.com/vuejs-translations/guidelines/blob/main/README.md), [ISO 639-1 language codes](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) do not distinguish between simplified and traditional Chinese.

Therefore, we plan to adopt the [IETF BCP 47 language tag](https://en.wikipedia.org/wiki/IETF_language_tag),

> zh-Hans and zh-Hant should be used instead of zh-CN/zh-SG/zh-MY and zh-TW/zh-HK/zh-MO.

**Comment by xiaodong2008** [comment]
@Jinjiang The translation is almost complete [here](https://github.com/xiaodong2008/docs-zh-hk), could you assist in transferring this to the organisation?

**Reply by Jinjiang** [reply]
Congrats. Sure. According to the guidelines, please make sure ahead the repo has been detached before the transfer. Thanks.

**Reply by sxzz** [reply]
You can detach the fork via GitHub support https://support.github.com/contact?flow=detach_fork&subject=Detach%20Fork&tags=rr-forks and wait few hours

**Reply by xiaodong2008** [reply]
@Jinjiang repo has been detached and transferred to [vuejs-translations/docs-zh-hk](https://github.com/vuejs-translations/docs-zh-hk)

**Reply by Jinjiang** [reply]
Cool!

The testing domain is working now https://docs-zh-hk.vercel.app/
And you will be noticed once the official one being available.

Cheers.

**Reply by xiaodong2008** [reply]
@Jinjiang 
That's great, I can now see the deployment on https://docs-zh-hk.vercel.app/
Btw, can you also give me access to the repo of zh-cn?

**Reply by Jinjiang** [reply]
May I know why you need the access to zh-cn please?

**Reply by xiaodong2008** [reply]
Ah sure, just want to help maintain zh-cn, this is not too related to this conversation, but I don’t have your contact information so I'm just pinging you here😉

**Reply by Jinjiang** [reply]
Understood. No problem. Done.

**Reply by Jinjiang** [reply]
https://zh-hk.vuejs.org/ is available now

**Reply by xiaodong2008** [reply]
Will hk.vuejs.org be better?

**Reply by Jinjiang** [reply]
We follow the IETF language tag, in which zh-hk is the valid one. Thanks for asking.

**Reply by xiaodong2008** [reply]
There's a question, I'm gonna add the link to the navbar and also the `Translations` page, but where should I place it?

After the Simplified Chinese or just the last place on the list?

**Reply by Jinjiang** [reply]
I see. Good question. I suggest putting at the bottom for now since it's the ordinary way we do. Then maybe by chance we can rearrange the list necessarily.
